hera_mc/watch_dog.py

```diff
@@ -172,6 +172,9 @@ def node_temperature(
 
         db = mc.connect_to_mc_db(None)
         session = db.sessionmaker()
+        close_session_when_done = True
```

Comment:
we want this to be tested on site as well.

Reply:
I think I'll need some assistance in these site test comments.